chasemapper/gpsd.py

```diff
@@ -426,8 +427,8 @@ def print_dict(data):
         format="%(asctime)s %(levelname)s:%(message)s", level=logging.DEBUG
     )
 
-    _gpsd = GPSDAdaptor(callback=print_dict)
-    time.sleep(30)
+    _gpsd = GPSDAdaptor(callback=print_dict, hostname="172.17.0.3")
```

Comment:
This should be configurable

Reply:
Yeah this was just a test script for debugging. I've made it take that hostname as an argument.